converter.py

```diff
@@ -323,7 +323,3 @@ def formatter(self):
             
```

Comment:
Before the first merge, the main branch should be empty. The main branch is a stable version, the dev branch is for development(for developing features).